xds/src/main/java/io/grpc/xds/internal/sds/SdsProtocolNegotiators.java

```diff
@@ -60,7 +60,7 @@ private SdsProtocolNegotiators() {
 
   private static final Logger logger = Logger.getLogger(SdsProtocolNegotiators.class.getName());
 
-  private static final AsciiString SCHEME = AsciiString.of("https");
+  private static final AsciiString SCHEME = AsciiString.of("http");
```

Comment:
IIRC there was a test checking scheme http if plaintext. the test should be fixed.

Reply:
There was/is no test - otherwise the build would have failed. Do you want me to add a test - not sure if we achieve anything with the test.

Reply:
https://github.com/grpc/grpc-java/blob/master/xds/src/test/java/io/grpc/xds/internal/sds/SdsProtocolNegotiatorsTest.java#L288

that test, it was checking "http" vs "https". now everything is "http". this test is no longer relevant and should be fixed.

Reply:
Yes, you are right, may bad! I will have to modify the test to detect plaintext vs Sds PN and let me think about it.

Reply:
With scheme being the same the only other thing to check is the class-name of the returned `protocolNegotiator` it should match `...ServerPlaintextNegotiator` or should have `plaintext` in it. Do you have a better suggestion?

Reply:
yeah there isn't a good way to test this other than using name of the class. i think it is fine to check the class name (`ServerPlaintextNegotiator`).